docs/mobile-apps/automated-testing/appium/real-devices.md

```diff
@@ -39,6 +39,83 @@ The following application file types are supported for real device tests:
+
+Since W3C and JWP use different formats for specifying your test capabilities, it is important to make sure you configure your tests accurately so your intended protocol is followed and your settings are applied correctly.
+
+### How Sauce Labs Determines Your Protocol
+
+When Sauce Labs executes your test configuration, it looks for the presence of certain indicators in the session creation request to determine whether it should apply the JWT or W3C protocol. The following table outlines how Sauce Labs evaluates your creation request.
+
+|Indicator|Determination|
+|:---|:---|
+|`sauce:options` node is present within the capabilities node.|W3C|
+|`desiredCapabilities` node is absent.|W3C|
+|`desiredCapabilities` node is present AND `sauce:options` node is absent.|JWP|
+
+### Examples of JWP and W3C Configurations
+
+The main difference between JWP and W3C is the format required to specify your test capabilities. In JWP, all capabilities are defined within the `desiredCapabilities` node, while in W3C, all capabilities are defined under the `capabilities` node within the `firstMatch` property. Each capability uses a namespace to indicate one of the following three categorizations:
+
+* WebDriver standard capabilities have no namespace
+* Appium specific capabilities require the prefix `appium:`
+* Sauce Labs custom capabilities are set within the `sauce:options` node
+
+The following examples illustrate this difference in the respective specifications.
+
+<Tabs
+  groupId="protocol"
+  defaultValue="jwp"
+  values={[
+    {label: 'JWP', value: 'jwp'},
+    {label: 'W3C', value: 'w3c'},
+  ]}>
+
+<TabItem value="jwp">
+
+```
+"desiredCapabilities": {
+    "platformName" : "android";
+    "app" : "./apps/mda-1.0.6-8.apk";
+    “deviceName" : "Samsung.*Galaxy.*”;
+    “orientation” : “portrait”;
+    “platformVersion" : "8.1";
+    "appiumVersion" : "1.21.0";
+ }
+```
+
+</TabItem>
+<TabItem value="w3c">
+
+```
+"capabilities": {
+    "firstMatch": [
+      {
+        "platformName" : "android",                    #standard capability
+        "appium:app" : "./apps/mda-1.0.6-8.apk";       #Appium capabilities
+        “appium:deviceName" : "Samsung.*Galaxy.*”;
+        “appium:orientation” : “portrait”;
+        “appium:platformVersion" : "8.1";
+        "sauce:options" : {                           #Sauce custom capabilities
+           "appiumVersion" : "1.21.0";
+           "sessionCreationRetry" : "2",
+           "sessionCreationTimeout" : "300000",
+           "name" : "MobileWebsiteTest (w3c)"
+        }
+      }
+    ]
+}
+```
+
+</TabItem>
+</Tabs>
+
+:::note
+You can avoid having to add the `appium:` prefix to Appium specific capabilities by upgrading your Appium client library to a version that automatically applies the prefix or installing the relaxed-caps plugin. Learn more in [Appium’s 2.0 Migration Capability Prefixes documentation](https://appiumpro.com/editions/123-migrating-to-appium-20-part-1-capability-prefixes).
```

Comment:
Appiumpro.com is a project by HeadSpin, which is a competitor. I'm not comfortable with referencing it here. I recommend to remove this link and the reference to it for now and build the docs with all the details on our Sauce docs instead. If this is something we can't do by December 1st, that's ok but we should prioritize it, as long as it brings value to users.

Reply:
Hi @linafarid  -- Oh wow. That is a big oops. I did not realize that is a competitor!  I think Abdulla or Ricardo pointed me to this site, when I was editing the KB article. We should reach out to Max because this link is also in that article, which doesn't look good for our support docs to be pointing at a competitor.